src/FtpExtractor.php

```diff
@@ -50,6 +51,21 @@ public function __construct(bool $onlyNewFiles, FtpFilesystem $ftpFs, LoggerInte
 
     public function copyFiles(string $sourcePath, string $destinationPath, FileStateRegistry $registry): int
     {
+        try {
+            /** @var AbstractFtpAdapter $adapter */
```

Comment:
to me teda neni jasny jaktoze to predtim fungovalo

Reply:
Ten filesystem, co tomu předáváš je obecný. Na tom voláš prostě věci jako na jakýmkoli jiným FS. Jenže já jsem si potřeboval sahnout přímo na ten ftp adapter, abych přímo zkusil to připojení.